### PR TITLE
Add alert UpperCase letters form

### PR DESCRIPTION
**Pull request by J2ZROMERO**
> Add alert email form:
- The email input just allows LowerCase letter.
- If there are UpperCase letters the form does not submit.
- Form desktop and mobile versions have the alert.

### CODE REVIEW

**Review by frank1738 — COMMENTED** [review]
Hi @J2ZROMERO , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have commits from all members in your pair programming group. Kindly note that this is a pair programming activity and all the members should commit to the project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by rloterh — COMMENTED** [review]
Hi Jhosep,
 
 While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
 
 ## Invalid Code Review Request
You have submitted a project, whereby ESLint is not properly setup, and hence, ESLint checks not showing on your Pull Request. Kindly ensure that **all** required linters are properly configured, and checks are passing. Kindly request for another code review when this is done.

**ESLint not showing:**
image
 
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
 
 Cheers and Happy coding!👏👏👏
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by J2ZROMERO** [comment]
- I have checked every file and all files are installed according to the next link: https://github.com/microverseinc/linters-config/tree/master/html-css-js
- All linkers are correctly installed

File: index.html [hunk 1]
```diff
@@ -380,20 +380,24 @@ <h2 class="opinion_me">I'm always interested in hearing about new
...
+                    
                 </li>
                 <li>
-                    <input class="form_input" name="email" type="email" placeholder="Email address" required>
+                    <input id="mail" class="form_input email_input" name="email" type="email" placeholder="Email address" required>
                 </li>
-                <li>
-                    <textarea class="form_input textarea" name="textarea" placeholder="Enter text here" maxlength="500"
+                <li class="error"> <span> The email must be in lowercase letters </span></li>
+                <li >
```
Comment (AnselemOdims) [inline index.html]: - [ ] Please ensure that the error message displays close to the button and also ensure that it does not disappear until the user corrects the email

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @J2ZROMERO , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by J2ZROMERO** [comment]
- Position alert changed close to the button
- Alert disappears after 3 seconds canceled, it changes when the user has correct the input

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @J2ZROMERO 

Your project is complete! Great job for making all the requested changes🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- Validations correctly implemented
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**